### PR TITLE
use docker volume for gitbase index

### PR DESCRIPTION
**Pull request by smacker**
Fix: #349 

I didn't delete data directory yet. It's unused now. Do you want me to delete it in this pr or in a separate one? Or just keep it as it is right now in case we will need it later?

### CODE REVIEW

**Comment by carlosms** [comment]
Could you please rebase on master, now that #351 has been merged?

**Comment by smacker** [comment]
rebased, all green.

File: cmd/srcd/daemon/daemon.go [hunk 1]
```diff
@@ -131,12 +129,7 @@ func start(workdir string) (*docker.Container, error) {
 }
 
 func setupDataDirectory(workdir, datadir string) error {
```
Comment (se7entyse7en) [inline cmd/srcd/daemon/daemon.go]: Can't we remove this func?

**Comment by smacker** [comment]
1. Removed everything related to datadir except `prune` cleanup.
2. Rebased on master

**Pull request merged**